src/features/emails/components/EmailEditor/index.tsx

```diff
@@ -61,7 +61,7 @@ const EmailEditor: FC<EmailEditorProps> = ({ email, onSave }) => {
   return (
     <Box display="flex" flexDirection="column" height="100%">
       {readOnly && (
-        <Alert severity="info" sx={{ marginBottom: 2 }}>
```

Comment:
Just as an FYI (and an opportunity to learn something): In MUI there is a shorthand for `marginBottom` called `mb`. The same is of course true for all other directions (e.g. `mr` for right). There's also `mx` and `my` for horizontal (both left and right) and vertical (both top and bottom) respectively.